### PR TITLE
Fix style

### PR DESCRIPTION
**Pull request by Kagami**
- Remove non-existing Conversations issue
- Add missed empty Bounty boxes
- Make link HTTPS

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
-[![OMEMO Logo](./images/omemo_logo.png)](http://omemo.top)
+[![OMEMO Logo](./images/omemo_logo.png)](https://omemo.top/)
 
 # Client Progress
 
 | **Client**  | Tracking Issue | Bounty | Work in Progress | Testing | Done |
 |-------------|----------------|--------|------------------|---------|------|
-| Conversations | [☑ #1](https://github.com/siacs/Conversations) |  | ☑ | ☑ | ☑ |
-| Gajim | [☑ #8161](https://dev.gajim.org/gajim/gajim/issues/8161) |  | ☑ | ☑ | ☑ |
+| Conversations | | ☐ | ☑ | ☑ | ☑ |
```
Comment (bascht) [inline index.md]: In all fairness – I think Conversations was the first client to implement OMEMO. I wanted to give it at least some credit. 

@iNPUTmice Shall we pick another issue or just leave it empty? 😸

**Comment by iNPUTmice** [comment]
Ich sehe kein Problem darin das 'Tracking Issue' feld einfach leer zu lassen.

**Comment by ReneVolution** [comment]
Can we add  ? 🌟

**Comment by Kagami** [comment]
To tracking issue column?

**Comment by ReneVolution** [comment]
Sorry, forgot to enclose it in <sarcastic /> tag 😊

**Comment by bascht** [comment]
@Kagami Sorry this issue was stale for so long and thanks again for helping out! :-)

**Pull request merged**